apps/transformers/lib/transformation_fields.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Transformers.TransformationFields do
+  alias Transformers
+
+  def fields_for("remove") do
+    Transformers.Remove.fields()
+  end
```

Comment:
It is possible, though I leave it to you if its a good idea, to reference a module by string:

```
module = String.to_existing_atom("Transformers.#{String.Capitalize(type)}")
module.fields()
```

Reply:
This would remove the need for this step in setting up a new transformation.

Reply:
That's good to know. I think I prefer the more explicit way of doing things, since adding an extra head on the hydra is very simple.